source/standards/vulnerability-disclosure.html.md.erb

```diff
@@ -0,0 +1,46 @@
+---
+title: Vulnerability Disclosure and security.txt
+last_reviewed_on: 2021-04-09
+review_in: 6 months
+---
+# <%= current_page.data.title %>
+
+## Vulnerability Disclosure
+
+In the Cabinet Office, including GDS, the [CDIO cyber security team] run a
+vulnerability disclosure programme with HackerOne and a third party to triage
+reports from security researchers.
+
+This isn't a sign-post for security reseachers to 'hack' our systems but we want
+to advocate secure disclosure so we can find out about issues and fix them
+before they cause a security incident.
+
+GOV.UK hosts the security policy:
+<https://www.gov.uk/help/report-vulnerability>
+
+## security.txt
+A `security.txt` file is a way of telling researchers how to get in contact with
+us. As per the current policy, we only accept reports from services that have a
+`security.txt` file pointing the [security policy].
+
+The file should be hosted at `https://{domain}/.well-known/security.txt`, use a
+`text/plain` content type and follow the [current security.txt guidance].
+
+We have a central deployment of the `security.txt` file so that we only have to
+keep one place up-to-date. The public [alphagov/security.txt] repo is where
+it's maintained.
+
+You should use <https://vdp.cabinetoffice.gov.uk/.well-known/security.txt>
+in either:
+
+- the origin for your site's `/.well-known/security.txt`
+- destination of 302 redirect for `/.well-known/security.txt`
+
+You may also do either (but only in addition to the `/.well-known/security.txt`
```

Comment:
I don't understand what 'you may also do either' refers to.

Reply:
Thanks for the review @cmcnallygds! I didn't like this bit, what I'm trying to say is either do the origin or a redirect for `/security.txt` but not instead of `/.well-known/security.txt`, only in addition